helpers/test-helper.js

```diff
@@ -260,13 +260,46 @@ function hexlifyAndPad(input) {
   return ethers.utils.hexZeroPad(ethers.utils.hexlify(i), 32);
 }
 
-export async function expectEvent(tx, eventName, args) {
-  const { logs } = await tx;
-  const event = logs.find((e) => e.event === eventName);
+export async function expectEvent(tx, nameOrSig, args) {
+  const re = /\((.*)\)/;
+  let event;
+  if (nameOrSig.match(re)) {
```

Comment:
Why is this being added?

Reply:
In the case of events being emitted in a transaction but not in the interface of the contract being called (as is done here for `createColony` on `colonyNetwork`, but checking to see `ColonyMetadata` has been emitted during the transaction), this additional functionality means we can specify the full signature of the event rather than just the name to check it has been emitted.

This specific line is checking for `()` in `nameOrSig` to work out if the full signature or just the name has been passed.